frontend/dockerfile/builder/build.go

```diff
@@ -324,6 +325,13 @@ func Build(ctx context.Context, c client.Client) (*client.Result, error) {
 				return nil, wrapSource(err, sourceMap, loc)
 			}
 			return res, err
+		} else if cmdline, ok := opts[keySyntax]; ok {
```

Comment:
wondering if build arg should override the value in the dockerfile? @AkihiroSuda

Reply:
yes, it should

Reply:
ok, it's done